public/sass/pages/_explore.scss

```
@@ -31,20 +31,24 @@
   flex-flow: row wrap;
   justify-content: flex-start;
   height: auto;
-  padding: 0 $dashboard-padding 0 50px;
+  padding: 0 $dashboard-padding 0 $dashboard-padding;
```

Small nitpick, could be shortened to:
```suggestion
  padding: 0 $dashboard-padding;
```